phenotrex/transforms/annotation.py

```diff
@@ -65,40 +67,58 @@ def fastas_to_grs(
 
     :param fasta_files: a list of DNA and/or protein FASTA files to be converted into
                         GenotypeRecords.
+    :param confidence_threshold: The threshold of confidence above which to keep an output of
+                                 deepnog annotation.
     :param verb: Whether to display progress of annotation with tqdm.
     :param n_threads: Number of parallel threads. Default, use all available CPU cores.
     :returns: A list of GenotypeRecords corresponding with supplied FASTA files.
     """
     n_threads = min(os.cpu_count(), n_threads) if n_threads is not None else os.cpu_count()
-    with ProcessPoolExecutor(max_workers=n_threads) as executor:
-        if len(fasta_files) > 1 and verb:
-            annotations = tqdm(
-                executor.map(fasta_to_gr, fasta_files),
-                total=len(fasta_files),
-                desc='Annotating with DeepNOG',
-                unit='file'
-            )
-        else:
-            annotations = executor.map(fasta_to_gr, fasta_files)
-    return list(annotations)
-
-
-def fasta_to_gr(fasta_file: str, verb: bool = False) -> GenotypeRecord:
+    func = partial(fasta_to_gr, confidence_threshold=confidence_threshold)
+    if len(fasta_files) > 1:
+        annotations = process_map(
+            func,
+            fasta_files,
+            max_workers=n_threads,
+            total=len(fasta_files),
+            desc='Annotating with DeepNOG',
+            unit='file'
+        )
+    else:
+        annotations = list(map(func,  fasta_files))
+    return annotations
+
+
+def fasta_to_gr(
+    fasta_file: str, confidence_threshold: float = None, verb: bool = False
+) -> GenotypeRecord:
     """
     Given a fasta file, determine whether gene calling is required (DNA fasta) or if deepnog can be
     applied directly (protein fasta). If required, perform prodigal gene call and return
     GenotypeRecord (output of deepnog) of the file.
 
     :param fasta_file: A DNA or protein fasta file to be converted into GenotypeRecord.
+    :param confidence_threshold: The threshold of confidence above which to keep an output of
+                                 deepnog annotation.
```

Comment:
```suggestion
    :param confidence_threshold: Confidence threshold of deepnog annotations below which
                                 annotations will be discarded.
```
Simply the same as above